### DISCUSSION TITLE
Error with searchable syncer

### DISCUSSION THREAD

**Discussion by marisavenuto** [thoughtspot/cs_tools#89] [Q&A] [answered]
Hi! 

I have typically been using the command:

`cs_tools tools searchable gather --syncer csv://directory=$HOME/Desktop/TS_CS_Tools`

to extract metadata. It has been working fine then all of a sudden (same day as it was was once working) I get an error. 
Other users at my organization don't seem to have the issue. 

Can you help?

<img width="587" alt="image" src="https://github.com/thoughtspot/cs_tools/assets/144272225/59f222b2-6410-4efd-b231-150534b0e641">

**Comment by boonhapus** [comment, marked as answer]
Have we enabled orgs @marisavenuto ? The users that's tied to this config might have had their org context switched in the UI. ThoughtSpot will save the last known org context on logout.

**Reply by marisavenuto** [reply]
Yes 100% this was why! I perpetually had both my Primary and Secondary org open with an incognito window 😅 
No other users at the organization were in the Secondary org except me which is why it worked for them. 
Noted for the future!!! Thank you so much 😄

**Reply by boonhapus** [reply]
We'll be adding support for orgs in the config file here soon ™️ , so that should lessen this kind of thing in the future!
